### PR TITLE
Don't show Applab tutorial for students younger than 13 years

### PR DESCRIPTION
**Pull request by Erin007**
On /congrats when a signed-in student completes an Hour of Code, and we know they are younger than 13, we show them age-appropriate options, rather than promote the Applab tutorial. 

Students younger than 13 who complete Minecraft will now see: 
screen shot 2017-11-17 at 8 27 50 pm

pre2017 v. 2017 will still be customized in the last card. 

Students younger than 13 who complete any other tutorial will now see: 
screen shot 2017-11-17 at 8 19 15 pm

This change required passing userAge as a prop through the Congrats component and making a new MinecraftMarketing card (pictured above).

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/src/templates/StudentsBeyondHoc.jsx [hunk 1]
```diff
@@ -27,20 +27,26 @@ export default class StudentsBeyondHoc extends Component {
...
     const desktop = (responsive.isResponsiveCategoryActive('lg') || responsive.isResponsiveCategoryActive('md'));
 
     const headingStyle = desktop ? styles.heading : styles.mobileHeading;
 
     var specificCardSet;
     switch (true) {
+      case completedTutorialType === 'pre2017Minecraft' && isEnglish && under13:
+          specificCardSet = 'youngerThan13pre2017MinecraftCards';
+        break;
       case completedTutorialType === 'pre2017Minecraft' && isEnglish:
```
Comment (breville) [inline apps/src/templates/StudentsBeyondHoc.jsx:50]: I guess you could be explicit about `!under13` but that might not be necessary.

**Comment by caleybrock** [comment]
If this is the image they approved then 👍 but the Minecraft image feels bolder/sharper with the gray bar like the graphic they have in previous years.

File: apps/src/templates/StudentsBeyondHoc.story.js [hunk 2]
```diff
@@ -36,6 +36,23 @@ export default storybook => {
...
               isRtl={false}
               userType="student"
+              userAge={14}
+              isEnglish={true}
+            />
+          </Provider>
+        )
+      },
+      {
+        name: 'Other tutorial, English, student under 13',
```
Comment (caleybrock) [inline apps/src/templates/StudentsBeyondHoc.story.js:46]: 👏 thank you for keeping all these cases clear in storybook

File: dashboard/app/views/congrats/index.html.haml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 :ruby
+  require 'cdo/user_helpers'
   congrats_data = {current_user: current_user}
```
Comment (caleybrock) [inline dashboard/app/views/congrats/index.html.haml:3]: is age not included in the current_user object?

**Comment by caleybrock** [comment]
I think this will be a really positive change for users 💯

File: dashboard/app/views/congrats/index.html.haml (hunk 3 above)
Comment (Erin007) [reply dashboard/app/views/congrats/index.html.haml:3]: birthday is, but not age. Also, I learned from Poorva recently that birthday isn't really birthday, it's the date you made an account and the year of your birth calculated from the age associated with the account info at sign up. I just used the same helper that is used in the project gallery to determine age from birthday.

Comment (caleybrock) [reply dashboard/app/views/congrats/index.html.haml:3]: 👍 good to know

**Comment by tanyaparker** [comment]
fyi @poorvasingal

**Pull request merged**